rslearn/main.py

```diff
@@ -525,6 +542,12 @@ def dataset_ingest() -> None:
         prog="rslearn dataset ingest",
         description="rslearn dataset ingest: ingest items in retrieved data sources",
     )
+    parser.add_argument(
+        "--disabled-layers",
+        type=lambda s: s.split(","),
+        default="[]",
```

Comment:
Should the default here be empty string? I think the processing function should then return empty array if the input is empty string. Otherwise it works for this disabled layer since the layer "[]" probably doesn't exist but it would be confusing pattern. At least that is my understanding of what is happening here.

Reply:
Or maybe the default should be [] without the string.